### PR TITLE
Refresh managed apps

### PR DESCRIPTION
**Pull request by grobbie**
## Done

- Removed some out of date listings
- Replaced a listing

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- [List additional steps to QA the new features or prove the bug has been resolved]

## Issue / Card

APPSPM-85

### CODE REVIEW

File: templates/managed/apps/index.html [hunk 1]
```diff
@@ -105,7 +105,7 @@ <h1>Performant open source with Managed Apps</h1>
         </div>
         <div class="p-matrix__content u-align--center">
           <div class="p-matrix__desc">
-            <p class="p-muted-heading">HA-READY PROMETHEUS INSTANCES</p>
+            <p class="p-muted-heading">LOW LATENCY QUERIES FOR BIG DATA</p>
```
Comment (anthonydillon) [inline templates/managed/apps/index.html:108]: Could you sentence case this for better screen reader accessibility?

File: templates/managed/apps/index.html [hunk 2]
```diff
@@ -146,7 +146,7 @@ <h1>Performant open source with Managed Apps</h1>
         </div>
         <div class="p-matrix__content u-align--center">
           <div class="p-matrix__desc">
-            <p class="p-muted-heading" style="margin-left: 1rem">HIGH-PERFORMANCE ML CLUSTERS</p>
+            <p class="p-muted-heading" style="margin-left: 1rem">AI/MLOPS AT ANY SCALE</p>
```
Comment (anthonydillon) [inline templates/managed/apps/index.html:149]: Same here, please

File: templates/managed/apps/index.html [hunk 3]
```diff
@@ -172,115 +172,13 @@ <h1>Performant open source with Managed Apps</h1>
...
-          loading="lazy",
-          attrs={"style": "height: 80px; width: auto; margin-bottom: 1rem; margin-top:.8rem;"},
-          ) | safe
-        }}
-        </div>
+	</div>
         <div class="p-matrix__content u-align--center">
           <div class="p-matrix__desc">
-            <p class="p-muted-heading u-align--center">SCALABLE RELATIONAL DATABASE</p>
+            <p class="p-muted-heading">THE WEBSCALE DATABASE</p>
```
Comment (anthonydillon) [inline templates/managed/apps/index.html:181]: Same here please

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Done small a11y improvements

**Review by anthonydillon — COMMENTED** [review]
Could you please update the copy doc for this page please.
https://docs.google.com/document/u/1/d/1fLRWW-Gtslaia2ypSrLJVBxZVHo5FY5Q84o97Gs2-w4/mobilebasic

**Comment by grobbie** [comment]
Updated for a11y and updated the copy doc [here](https://docs.google.com/document/d/1fLRWW-Gtslaia2ypSrLJVBxZVHo5FY5Q84o97Gs2-w4/edit) - please check there isn't a duplicate

File: templates/managed/apps/index.html [hunk 4]
```diff
@@ -85,18 +85,18 @@ <h1>Performant open source with Managed Apps</h1>
...
-          alt="promethus",
-          width="144",
-          height="90",
+          url="https://assets.ubuntu.com/v1/11e66fc8-Opensearch_Logo.svg",
+          alt="OpenSearch",
+          width="125",
+          height="25",
           hi_def=True,
           loading="lazy",
           attrs={"style": "height: 80px; width: auto;"},
```
Comment (sowasred2012) [inline templates/managed/apps/index.html:102]: The image is rendering in a way that makes it look low res, this fixes it:
```suggestion
          alt="OpenSearch",
          width="200",
          height="40",
          hi_def=True,
          loading="lazy",
          attrs={"style": "margin-top:.8rem;"},
```

File: templates/managed/apps/index.html [hunk 5]
```diff
@@ -166,121 +165,19 @@ <h1>Performant open source with Managed Apps</h1>
         </div>
         <div class="p-matrix__content u-align--center">
           <div class="p-matrix__desc">
-            <p class="p-muted-heading">MANAGED RELATIONAL DATABASES</p>
+            <p class="p-muted-heading">Managed relational databases</p>
           </div>
         </div>
       </li>
       <li class="p-matrix__item u-align--center">
         <div style="min-width: 50%">
```
Comment (sowasred2012) [inline templates/managed/apps/index.html:173]: This "MongoDB CE" block looks odd without an image compared to the others, almost as if the image failed to load and the alt text is being display instead. Is there nothing we could use? If not, we could align it a little better with:

```suggestion
        <div style="min-width: 50%; margin-top: 2.4rem; margin-bottom: 2rem;">
```

**Review by sowasred2012 — CHANGES_REQUESTED** [review]
Looks good, just two bits of housekeeping.

**Review by sowasred2012 — APPROVED** [review]
LGTM

**Pull request merged**